test/basic016/expected

```diff
@@ -1 +1 @@
-Cycle detected in imports: CycleA.idr -> CycleB -> CycleC -> CycleA -> CycleB
+ Cycle detected in imports: CycleA.idr -> CycleB -> CycleC -> CycleA -> CycleB
```

Comment:
You should not change this PR now, but it would be cool if it highlighted the `import` statement at some point.

Reply:
OK, will look later.